fabfile/env/platforms.py

```diff
@@ -92,7 +92,7 @@ def make_ssh_url(serv, *args):
 env.KRAKEN_RABBITMQ_WRONG_PORT = 56722
 env.PUPPET_RESTART_DELAY = 30
 # max time (in s) that a kraken can take to restart
-env.KRAKEN_RESTART_DELAY = 90
+env.KRAKEN_RESTART_DELAY = 300  # 90
```

Comment:
I think you can remove the comment
```suggestion
env.KRAKEN_RESTART_DELAY = 300
```